ES/ESmy024.xml

```diff
@@ -57,7 +57,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <textLang mainLang="gez"/>
```

Comment:
in ESmy024 the collation has issues
```
<item xml:id="q7" n="6">
                                <num value="2">፪</num>
                                 <dim unit="leaf">8</dim>
                                 <locus from="41r" to="48v"/>
                                 s.l.: 3, stub after 5
                               </item>
```
a quire with 8 leaves cannot have just one single leaf. or none, or two.

Reply:
ok, corrected.